zigbee2mqtt/DOCS.md

```diff
@@ -1,3 +1,34 @@
+# Installation / Configuration after 1.25.0-1
+1.25.0-1 introduced some breaking changes. From this release, all configuration are done in the `$DATA_PATH/configuration.yaml`
+
+When installing/updating 1.25.0-1 you will se the following configuration:
+
+```data_path: /config/zigbee2mqtt
+socat:
+  enabled: false
+  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
+  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
+  options: '-d -d'
+  log: false
+mqtt: {}
+serial: {}
+```
+The ONLY thing you have to configure here is mqtt and serial.
+If you use Home Assistant mqtt server, you should use:
+
+```
+mqtt:
+  server: mqtt://core-mosquitto:1883
```

Comment:
I believe this can be left empty. (tested by installing the mosquitto addon and then start z2m addon with leaving this empty)

Reply:
Yup I can also confirm as per my last comment in this PR :)
The addon automatically finds the server, user and password from the HA mosquitto addon when this is left empty.